MANIFEST.in

```diff
@@ -1,2 +1,3 @@
 include libraries/*
 include schema/*
+include debian/changelog
```

Comment:
Should the code in `setup.py` perhaps fail if the file isn't present? Assuming we always have it either from git or the dist tarball.

Reply:
that would break building the snap, it is a larger refactor we can approach together with splitting out debian/changelog from the tree.